src/textual/widgets/_directory_tree.py

```diff
@@ -119,6 +158,16 @@ def process_label(self, label: TextType):
         return first_line
 
     def render_label(self, node: TreeNode[DirEntry], base_style: Style, style: Style):
```

Comment:
Type annotation is missing here, to go with the `Returns` in the docstring.

Reply:
Yep. Looks like it slipped through a previous review. @davep would you mind?